files/firstLaunch.js

```diff
@@ -207,6 +214,129 @@ module.exports = async () => {
         })
     }
 
+    async function runSpeedtest() {
+        // write config file in librespeed-cli folder
+        const configFile = `${process.cwd()}/files/librespeed-cli/config.json`
+        const config = [
+            {
+                "id": 1,
+                "name": "o!rdrFR",
+                "server": "https://st1.issou.best/",
+                "dlURL": "garbage.php",
+                "ulURL": "empty.php",
+                "pingURL": "empty.php",
+                "getIpURL": "getIP.php"
+            }
+        ]
+        fs.writeFileSync(configFile, JSON.stringify(config))
+
+        // Run speedtest
+        console.log("Running speedtest...")
+        const speedtest = spawn(`${process.cwd()}/files/librespeed-cli/librespeed-cli` + (process.platform === "win32" ? ".exe" : ""), ["--json", "--local-json", `${process.cwd()}/files/librespeed-cli/config.json`])
+        speedtest.stdout.setEncoding("utf8")
+        speedtest.stdout.on("data", async data => {
+            const parsedData = JSON.parse(data)
+
+            console.log(`Download: ${parsedData[0].download} Mbps`)
+            console.log(`Upload: ${parsedData[0].upload} Mbps`)
+
+            // prompt user if they want to continue
+            const cont = await inquirer.prompt([
+                {
+                    type: "confirm",
+                    name: "continue",
+                    message: "Do you want to continue?",
+                    default: true
+                }
+            ])
+
+            if (cont.continue) {
+                chooseRenderingType()
+            } else {
+                exit()
+            }
+        })
+
+        speedtest.stderr.setEncoding("utf8")
+        speedtest.stderr.on("data", data => {
+            console.log('There was an error performing the speedtest, skipping...')
+            chooseRenderingType()
+        })
+    }
+
+    async function downloadLibrespeedCli() {
+        // set different links for different platforms (windows, linux, mac)
+        let link;
```

Comment:
use http://dl.issou.best/ordr/librespeed-cli-win.zip on windows and http://dl.issou.best/ordr/librespeed-cli-linux.zip on linux